cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/sv/CVRSvDataReader.java

```diff
@@ -111,12 +112,17 @@ public void open(ExecutionContext ec) throws ItemStreamException {
             }
             reader.close();
         }
-        
         for (CVRMergedResult result : cvrData.getResults()) {
             String sampleId = result.getMetaData().getDmpSampleId();
             List<CVRSvVariant> variants = result.getSvVariants();
             for (CVRSvVariant variant : variants) {
-                CVRSvRecord record = new CVRSvRecord(variant, sampleId);
+                CVRSvRecord record = svUtilities.makeCvrSvRecordFromCvrSvVariant(variant, sampleId);
```

Comment:
same comment as below about calling these populate `field` functions from inside the `.makeCvrSvRecordFromCvrSvVariant` since they will always be called together